FWCore/Integration/test/testMissingDictionaryCUDAChecking.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+function die {
+  echo -e "$1"
+  echo "exit status $2"
+  exit $2
+}
+
+LOCAL_TEST_DIR=${SCRAM_TEST_PATH:-$CMSSW_BASE/src/FWCore/Integration/test}
+
+cmsRun ${LOCAL_TEST_DIR}/testMissingDictionaryCUDAChecking_cfg.py >& testMissingDictionaryCUDAChecking.log
+grep -q "An exception of category 'DictionaryNotFound' occurred" testMissingDictionaryCUDAChecking.log || die "Cannot find the following string in the exception message:\nAn exception of category 'DictionaryNotFound' occurred" $?
```

Comment:
I'd add a check here that the `cmsRun` exited with a non-zero exit code.

Reply:
Sure.